libs/error-analysis/src/lib/ErrorAnalysisDashboard/ErrorAnalysisDashboard.tsx

```diff
@@ -380,6 +381,8 @@ export class ErrorAnalysisDashboard extends React.PureComponent<
             viewType={this.state.viewType}
             activeGlobalTab={this.state.activeGlobalTab}
             activePredictionTab={this.state.predictionTab}
+            showMessageBar={this.state.showMessageBar}
+            closeMessageBar={this.closeMessageBar.bind(this)}
```

Comment:
we should avoid using bind in render, it will create a new instance everytime render get called.
instead change the function defination to be arrow function could be a good practice.

Reply:
done!